content/blog/2021-03-26-cdn-demo.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Content Delivery Network with Zenoh"
+date: 2021-03-27
+menu: "blog"
+weight: 300
+description: "27 March 2021 -- Paris."
+draft: false
+---
+
+## Why a Content Delivery Network?
+A Content Delivery Network (CDN) is a popular solution based on geographically distributed servers that work together by collecting the data to the user’s nearest node. Edge servers, located close to the end users, are the base infrastructure a CDN is built upon.
```

Comment:
Instead of "by collecting the data..." I'd say: " by storing the data " or perhaps "by caching the data"